doc/classes/EditorSettings.xml

```diff
@@ -734,6 +827,10 @@
 			Depending on the platform and used renderer, the engine will fall back to [b]Enabled[/b] if the desired mode is not supported.
 			[b]Note:[/b] V-Sync modes other than [b]Enabled[/b] are only supported in the Forward+ and Mobile rendering methods, not Compatibility.
 		</member>
+		<member name="interface/editors/derive_script_globals_by_name" type="bool" setter="" getter="">
+		</member>
```

Comment:
```suggestion
		<member name="interface/editors/derive_script_globals_by_name" type="bool" setter="" getter="">
			If [code]true[/code], when extending a custom script and it has a global name, the Inherits field of script creation dialog will be filled with the global name instead of script path.
		</member>
```